src/config/index.js

```diff
@@ -19,9 +19,20 @@ const firebaseConfig = {
   measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
 };
 
+console.log("firebaseConfig", firebaseConfig);
```

Comment:
do not put console log codes in commits, if you use console log for debugging remove them after.

Reply:
Yeah I used it for debugging, The changes updated.